components/tag/stories/template.js

```diff
@@ -35,32 +40,50 @@ export const Template = ({
 					typeof size !== "undefined",
 				"is-emphasized": isEmphasized,
 				"is-disabled": isDisabled,
-				"is-invalid": isInvalid,
 				"is-selected": isSelected,
+				"is-hover": isHovered,
+				"is-focus-visible": isFocused,
+				"is-active": isActive,
+				"is-readOnly": isReadOnly,
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
 			id=${ifDefined(id)}
 			tabindex=${isDisabled ? "-1" : "0"}
 			style=${styleMap(customStyles)}
+			@click=${function() {
+				if (isReadOnly || isDisabled) return;
+				updateArgs({ isSelected: !isSelected });
+			}}
+			@focusin=${function() {
+				if (isReadOnly || isDisabled) return;
+				updateArgs({ isFocused: true });
+			}}
+			@focusout=${function() {
+				updateArgs({ isFocused: false });
+			}}
```

Comment:
I'd recommend removing this, as the focus events are not the same behavior as `:focus-visible`. So currently if you click on the tag in the Default story, the focus indicator will appear when it shouldn't.

Reply:
Ok! I'm using the keyboard focused instead of focused control now as well. I think that I'd thought it would be nice to see the control toggling when you keyboard focus, but that does have unintended side effects.